src/common/redis_module/ray_redis_module.cc

```diff
@@ -908,6 +910,74 @@ int TableTestAndUpdate_RedisCommand(RedisModuleCtx *ctx,
   return result;
 }
 
+/* send notification to the subscribers */
+int sendObjectNotification(RedisModuleCtx *ctx,
```

Comment:
Is it necessary to introduce this method? I think all we need to do is call `PublishObjectNotification` from inside of `ObjectTableRemove_RedisCommand`, right?

Reply:
We need to call sendObjectNotification in ObjectTableAdd_RedisCommand and ObjectTableRemove_RedisCommand. So I pull out this code from ObjectTableAdd_RedisCommand, and add the function sendObjectNotification. We'll create anohter pull request to fix the perf problem of this funcion(i.e. the comment 2 of @raulchen ).